tools/doc/package.json

```diff
@@ -7,7 +7,15 @@
     "node": ">=6"
   },
   "dependencies": {
-    "marked": "^0.3.5"
+    "jsdom": "^11.11.0",
```

Comment:
Is this dependency intended for this PR?

Reply:
Good catch!  Fixed.